packages/calcite-components/vite.config.ts

```diff
@@ -43,6 +45,15 @@ export default defineConfig({
         },
       },
     }),
+    replace({
```

Comment:
(optional) vite provides env variables support without a plugin, so if you don't need extra customization options you can use that to keep build config simpler and performance better: https://vite.dev/guide/env-and-mode